backend/apps/audio/main.py

```diff
@@ -43,6 +43,8 @@
     DEVICE_TYPE,
     AUDIO_OPENAI_API_BASE_URL,
     AUDIO_OPENAI_API_KEY,
+    AUDIO_OPENAI_API_MODEL,
+    AUDIO_OPENAI_API_SPEAKER,
```

Comment:
Should be renamed to `AUDIO_OPENAI_API_VOICE`

Reply:
ok